app/models/barcode_sheet.rb

```diff
@@ -44,6 +44,10 @@ def print_to_sprint
     raise(PrintError, 'There was an error sending a print request to SPrint') unless response.code == '200'
   end
 
+  def sprint_labels
```

Comment:
Don't think this is being called from anywhere? Does it want to be used in `print_to_sprint` ?

Reply:
How come extra_label in needs to be ignored?

Reply:
> How come extra_label in needs to be ignored?

Is extra label used for tag plates? The issue is that at the moment the labels are blank because they are nested in main_label. I spent most of yesterday on this and when I checked main_label was the only one being used for tag plates?